soundata/datasets/urbansound8k.py

```diff
@@ -0,0 +1,355 @@
+            [classID] = a numeric identifier of the sound class (see description of classID below for further details)
+            [occurrenceID] = a numeric identifier to distinguish different occurrences of the sound within the original recording
+            [sliceID] = a numeric identifier to distinguish different slices taken from the same occurrence
+        fsID (str): The Freesound ID of the recording from which this excerpt (slice) is taken
+        start (float): The start time in seconds of the slice in the original Freesound recording
+        end (float): The end time in seconds of slice in the original Freesound recording
+        salience (int): A (subjective) salience rating of the sound. 1 = foreground, 2 = background.
+        fold (int): The fold number (1-10) to which this file has been allocated.
+        classID (int): A numeric identifier of the sound class:
+            0 = air_conditioner
+            1 = car_horn
+            2 = children_playing
+            3 = dog_bark
+            4 = drilling
+            5 = engine_idling
+            6 = gun_shot
+            7 = jackhammer
+            8 = siren
+            9 = street_music
+        class (str): The class name: air_conditioner, car_horn, children_playing, dog_bark, drilling, engine_idling, gun_shot, jackhammer, siren, street_music.
+        clip_id (str): clip id
+
+    """
+
+    def __init__(
+        self,
+        clip_id,
+        data_home,
+        dataset_name,
+        index,
+        metadata,
+    ):
+        super().__init__(
+            clip_id,
+            data_home,
+            dataset_name,
+            index,
+            metadata,
+        )
+
+        self.audio_path = self.get_path("audio")
+
+    @property
+    def audio(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The clip's audio
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    @property
+    def slice_file_name(self):
+        return self._clip_metadata.get("slice_file_name")
+
+    @property
+    def freesound_id(self):
+        return self._clip_metadata.get("freesound_id")
+
+    @property
+    def start_time(self):
+        return self._clip_metadata.get("start_time")
+
+    @property
+    def end_time(self):
+        return self._clip_metadata.get("end_time")
+
+    @property
+    def salience(self):
+        return self._clip_metadata.get("salience")
+
+    @property
+    def fold(self):
+        return self._clip_metadata.get("fold")
+
+    @property
+    def class_id(self):
+        return self._clip_metadata.get("class_id")
+
+    @property
+    def class_label(self):
+        return self._clip_metadata.get("class_label")
+
+    @property
+    def tags(self):
+        return annotations.Tags([self._clip_metadata.get("class_label")], [1.0])
+
+    def to_jams(self):
```

Comment:
Do we need jams at all in soundata?

Reply:
I think we do, it's used for a variety of DCASE dataset (e.g. augmented urbansound8k, URBAN-SED, FUSS). In fact maybe JAMS is being used more for DCASE research right now than for MIR :sweat_smile:

Reply:
Also everything generated by Scaper...

Reply:
Oh! Well, that's great then that we have the `jams_converter` implemented. We might have to give it a look to adapt it more for how it's been used in DCASE

Reply:
👍